lib/plugin/instance.rb

```diff
@@ -1177,17 +1177,6 @@ def register_bookmarkable(klass)
     DiscoursePluginRegistry.register_bookmarkable(RegisteredBookmarkable.new(klass), self)
   end
 
-  ##
-  # Register an object that inherits from [Summarization::Base], which provides a way
-  # to summarize content. Staff can select which strategy to use
-  # through the `summarization_strategy` setting.
-  def register_summarization_strategy(strategy)
```

Comment:
Let's just deprecate the method for now and see if it's enough to make the build green. We can remove post-move